### ISSUE TITLE
Enforce software versions

### ISSUE THREAD

**Issue by mairindeith** [mairindeith/fbwR#5]
Bugs with `dplyr` when not using the most up to date version of R. Include tests for this.

**Comment by mkoohafkan** [comment]
Is this an issue with the R version or the dplyr package version? Either way, this can be solved by specifying the minimum R and/or dplyr version in DESCRIPTION, i e.

```
Depends:
    R (>= 4.2)
Imports:
    dplyr (>= 1.1),
    ...
```

**Comment by mairindeith** [comment]
Thanks @mkoohafkan - I appreciate you chiming in! 

Just FYI, Alejandro and I are using GitHub issues as to-do trackers using the Projects tab, so there are issues here that we are in progress of working on, but have here for our own organization. We'll be writing issues that are simple fixes, so don't feel you have to address these issues.

**Comment by mairindeith** [comment]
Closed with commit [435245c](https://github.com/mairindeith/fbwR/commit/435245c8dc667d0192ae337cdf83138aaac94de1).
